_tests/apps_test.go

```diff
@@ -38,42 +34,40 @@ var _ = Describe("Apps", func() {
 		})
 
 		It("can create an app", func() {
-			sess, err := start("deis apps:create %s", app1Name)
-			Expect(err).To(BeNil())
-			Eventually(sess).Should(gexec.Exit(0))
-			Eventually(sess).Should(gbytes.Say("Creating Application... done, created %s", app1Name))
-			Eventually(sess).Should(gbytes.Say("Git remote deis added"))
-			Eventually(sess).Should(gbytes.Say("remote available at "))
-
-			sess, err = start("deis apps:destroy --confirm=%s", app1Name)
-			Expect(err).To(BeNil())
-			Eventually(sess).Should(gexec.Exit(0))
-			Eventually(sess).Should(gbytes.Say("Destroying %s...", app1Name))
-			Eventually(sess).Should(gbytes.Say("done in "))
-			Eventually(sess).Should(gbytes.Say("Git remote deis removed"))
+			output, err := execute("deis apps:create %s", appName)
```

Comment:
Can this still go back to the gexec based flow? Not blocking for this PR, in the interest of getting it merged.